src/FileSystem.cpp

```diff
@@ -0,0 +1,70 @@
+/*
+ *  HDRMerge - HDR exposure merging software.
+ *  Copyright 2012 Javier Celaya

```

Comment:
@Hombre57 you authored the file, so you have the copyright and no one else. You can licence it under the GPL, but you're still the copyright holder, so this line should be:
`Copyright 2018 <your name> <contact details>`

> This involves adding two elements to each source file of your program: a copyright notice (such as “Copyright 1999 Terry Jones”), and a statement of copying permission, saying that the program is distributed under the terms of the GNU General Public License (or the Lesser GPL, or the Affero GPL).

https://www.gnu.org/licenses/gpl-howto.en.html
http://gplv3.fsf.org/wiki/index.php/Howto